src/batch-client/index.ts

```diff
@@ -1233,6 +1235,15 @@ export class ZimbraBatchClient {
 			singleRequest: true
 		}).then(Boolean);
 
+	public saveDocument = (doc: SaveDocumentInput) =>
+		this.jsonRequest({
+			name: 'SaveDocument',
+			body: {
+				doc: denormalize(SaveDocument)(doc)
```

Comment:
we should have doc key also part of grapqhl
check AccountRights entity

Reply:
Is there any scenario where we have to pass list of doc ?

Reply:
no, we are passing single doc to be saved at a time each doc will be saved seperately in a request , we earlier had batch request which is converted to single request for handling same name files

Reply:
Done